blocks/hero-banner/hero-banner.css

```diff
@@ -0,0 +1,153 @@
+main .section.hero-banner-container {
+    max-width: 100%;
+    padding: 0;
+    margin-bottom: 0;
+}
+
+main .section.hero-banner-container>div {
+    margin: 0;
+    padding: 0;
+    max-width: unset;
+}
+
+
+.hero-banner.block {
```

Comment:
Minor: Is .block necessary here - not sure that level of specificity is needed

Reply:
ya..this is a convention we followed in my previous project.. to enforce specificity to that particular block.